packages/table/src/table.tsx

```diff
@@ -546,7 +546,7 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
                     quadrantRef={this.quadrantRefHandlers[QuadrantType.MAIN].quadrant}
                     quadrantType={QuadrantType.MAIN}
                     renderBody={this.renderBody}
-                    renderColumnHeader={this.renderColumnHeader}
+                    renderColumnHeader={this.renderMainQuadrantColumnHeader}
```

Comment:
/looks at PR. 

Oh yikes, that is a significant change. 

Either way is probably fine? I can handle doing that though it'll be annoying, but it's probably easier for my stuff to go in first? Not sure though.